.travis.yml

```diff
@@ -11,11 +11,10 @@ env:
     - "JRUBY_OPTS=--dev"
 
 rvm:
-  - 2.1.10
-  - 2.2.9
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
+  - 2.3.8
+  - 2.4.5
+  - 2.5.5
+  - 2.6.2
   - jruby-9.1.15.0
```

Comment:
Why not update JRuby as well?

Reply:
I'm not familiar with JRuby...
I'll add 9.2.6.0 that newest version of JRuby.